project-base/storefront/public/locales/cs/common.json

```diff
@@ -89,9 +89,11 @@
     "Customer information": "Informace o zákazníkovi",
     "Customized E-shop by": "E-shop na míru od",
     "Czechia": "Česko",
+    "date of order": "datum objednání",
     "Decrease": "Snížit",
     "Delete all": "Smazat vše",
     "Delete all from wishlist": "Smazat vše z oblíbených produktů",
+    "delivery": "doprava",
```

Comment:
I think it should be "transport", at least that's how we use it everywhere across the code. "Delivery" is more like "doručení" in our context. We could of course discuss if that is correct or not, but for the sake of keeping it the same, I would rewrite this to transport

Reply:
Fixed